docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+1. Repeat the process to add further sections.
+
+!!! warning
+    A name can only be assigned once, duplicate names of several sections are not possible.
+    We recommend using the approximate compass direction of the geographical location (e.g. north, north-east) as the name of the section.
+
+![Add Sections](usage-ui/add-section.png)
+
+#### Change the geometry of a section
+
+The geometry of sections that have already been created can be changed using the following procedure:
+
+1. First select the section to be changed in the list.
+1. Then start the change mode by left-clicking on the *Edit* button. The supporting points are now visible.
+1. Support points can now be selected by left-clicking and reset as in add mode. The + button can be used to add another support point.
+
+The selected point “sticks” to the mouse pointer until the left mouse button is pressed.
+The old geometry is displayed as a dashed reference.
+
+1. Right-click to exit change mode and save the changes.
+1. Pressing the Escape button exits the change mode without saving the changes to the geometry.
+
+#### Change attributes of a section
+
+The attributes (name and offset) of a section can be changed by clicking on the *Properties* button.
+To do this, the desired section must first be selected from the list.
+
+#### Remove a section
+
+A section can be removed by clicking on the *Remove* button.
+To do this, the desired section must first be selected in the list and must not be assigned to a flow.
+
+### Flows
+
+A flow always consists of exactly two sections: a start section and an end section.
+To edit flows, the *Flows* tab must first be selected in the Sections/Flows section.
+
+#### Add Flows
+
+A new flow is added in the following steps:
+
+1. Left-click on the *Add* button. A pop-up window opens. The appropriate sections can be selected from the drop-down menu.
+1. The start point of the flow is defined as the first section.
+1. The endpoint of the flow is defined as the second section.
+1. The name is automatically set in the *Name* field after selecting the two sections. This can be changed as required. The name is reused in the analysis.
+
+!!! warning
+    A name can only be assigned once, duplicate names of several flows are not possible.
+
+    Alternatively, flows can also be generated automatically using the *Generate* button.
+    Flows are then generated for all possible combinations of start and end sections.
+    The standard naming convention is `Name_Startsection --> Name_Endsection`.
+
+![Add Flows](usage-ui/add-flow.png)
+
+#### Edit Flows
+
+To edit a flow, it must first be selected in the list.
+Then left-click on the *Properties* button to open the same pop-up window as when creating a flow.
+The desired changes can now be made.
+
+#### Remove Flows
+
+To remove a flow, it must first be selected in the list.
+Left-click on the *Remove* button to remove the selected flow.
+
+## Visualization layers
```

Comment:
Should we put "visualization layers" in front of "definition of sections and flows"? First you want to see some tracks. And you need that to properly configure sections...